components/webui/imports/api/ingestion/server/CompressionDbManager.js

```diff
@@ -23,12 +23,12 @@ class CompressionDbManager {
      * Retrieves the last `limit` number of jobs and the ones with the given
```

Comment:
Let's update this docstring
```suggestion
     * Retrieves compression jobs that are updated on or after a specific time.
```